services/task-svc/internal/room/room.go

```diff
@@ -116,5 +116,13 @@ func (ServiceServer) AddBedsToRoom(ctx context.Context, req *api.AddBedsToRoomRe
 		Int("beds", len(beds)).
 		Msg("beds added")
 
-	return &api.AddBedsToRoomResponse{}, err
+	return &pb.AddBedsToRoomResponse{}, err
+}
+
+func BedsToBedsOfRoom(beds []models.Bed) []*pb.BedOfRoom {
```

Comment:
In my last pull request, this was moved to `api/`. Any reason why this is added again to `room/`?

Reply:
I believe we can't move it to `gen`, as `task-svc` depends on `gen` and this would introduce an import of `task-svc` in `gen`. Cyclic Dependency Graphs don't compile in go AFAIK.